rotkehlchen/tests/api/test_blockchain.py

```diff
@@ -452,16 +441,11 @@ def test_add_blockchain_accounts(
         })
         if async_query:
             task_id = assert_ok_async_response(response)
-            outcome = wait_for_async_task_with_result(rotkehlchen_api_server, task_id)
+            wait_for_async_task(rotkehlchen_api_server, task_id)
         else:
-            outcome = assert_proper_response_with_result(response)
+            assert_proper_response(response)
 
-    assert_btc_balances_result(
```

Comment:
Hm, I don't remember what I meant by linking another test, but aren't btc balances in this test still checked in this test (at line 491) ?

Reply:
The test you removed checks that only btc balances are queried. Down the line it checks that all balances are queried. 

So what it tests is the difference of the functionality of those 2.

Reply:
Hm... But take a look here:
1. Check that I removed is https://github.com/rotki/rotki/blob/develop/rotkehlchen/tests/api/test_blockchain.py#L459
2. Check that is checking balances for all blockchains https://github.com/rotki/rotki/blob/develop/rotkehlchen/tests/api/test_blockchain.py#L490

These checks look identical (and they also both have `also_eth=True` flag) which should mean that they are checking eth balances as well.

Or do I miss something?

Reply:
The checking function is the same. The "outcome" argument, what is being read is not. Since the first is the outcome of the "query only BTC" balances endpoint and the other is the outcome of query all balances.

But if this difference is being tested in some other test (perhaps in balances.py?) it should be fine to omit.